companion/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+  "id": "novastar-splicer",
+  "name": "novastar-splicer",
+  "shortname": "novastar-splicer",
```

Comment:
Just a reminder that the module itself needs renaming when this is merged, otherwise it will stop showing up on the website.

Reply:
OK, do I need to modify this? We hope that the merged module will be named "novastar-splicer"

Reply:
No, this is just to flag to make sure someone with suitable rights renames the repository when this is merged.